.github/workflows/check.yml

```diff
@@ -26,5 +42,7 @@ jobs:
         with:
           node-version-file: '.nvmrc'
           cache: 'npm'
+          cache-dependency-path: action/package-lock.json
```

Comment:
Is there a reason we're caching the action dependencies in a job that builds the server?

Reply:
was part of trying to fix the weird npm errors.